docs/pages/run/validator-management/vc-configuration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Stake with a Validator Client
+title: Starting a Validator Client
```

Comment:
need to come up with some better title, "Validator Client" is redundant here. I liked the previous "Configuration" more, or maybe "Usage" (not sure myself)

Reply:
For consistency with how we've named "Start a beacon node", I think we should just stick to "Starting a Validator Client." We can always follow up on a new PR in the future.